src/styles/blocks/contacts.scss

```diff
@@ -0,0 +1,51 @@
+.contacts {
+  margin-bottom: 120px;
+
+  &__title {
+    @include title;
+    margin-bottom: 48px;
+
+    @include onTablet {
+      font-size: 40px;
+      line-height: 40px;
+    }
+
+    @include onDesktop {
+      font-size: 48px;
+      line-height: 48px;
+    }
+  }
+
+  &__info {
+    &:first-child {
+      margin-top: 48px;
+
+      @include onTablet {
+        margin-top: 0;
+      }
+    }
+
+    &:not(:last-child) {
+      margin-bottom: 24px;
+    }
+
+    &-title {
+      margin-bottom: 8px;
+      font-size: 14px;
+      line-height: 20px;
+      color: $contactsTitleColor;
+    }
+
+    &-link {
```

Comment:
Use double underscores for nested elements instead of a single dash. Rename the class to '&__info-link'.